docs.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Docs
+permalink: /docs/
+---
+
+# User Guide
+[kanidm book](https://kanidm.github.io/kanidm/)
+
+
+# Developer Guide
+[guide for developers](https://github.com/kanidm/kanidm/blob/master/DEVELOPER_README.md)
```

Comment:
The developer guide is also in the book - https://kanidm.github.io/kanidm/DEVELOPER_README.html

Reply:
I changed it.